TRANSCRIPTION.md

```diff
@@ -75,27 +75,21 @@
 | ð | 3 | ✓ | ✓ |
 | õ | 3 | ✓ | ✓ |
 | ɾʰ | 3 | ✓ | ✓ |
-| aʰ | 2 | ? | ? |
 | aˀ | 2 | ✓ | ✓ |
 | hʲ | 2 | ✓ | ✓ |
 | z | 2 | ✓ | ✓ |
 | ãː | 2 | ✓ | ✓ |
 | ə̃ | 2 | ✓ | ✓ |
-| iʰ | 1 | ? | ? |
 | i̥ | 1 | ✓ | ✓ |
 | jʰ | 1 | ✓ | ✓ |
-| ka | 1 | ? | ? |
 | kw | 1 | ✓ | ✓ |
-| nⁱ | 1 | ? | ? |
 | r | 1 | ✓ | ✓ |
-| teː | 1 | ? | ? |
 | âː | 1 | ✓ | ✓ |
 | íː | 1 | ✓ | ✓ |
```

Comment:
Are you sure this is a vowel with high tone? Are those languages tonal?

Reply:
Some of the Baniwa languages are tonal, yes. Should I in any case map these to things like "â/a"?

Reply:
Yes, I think that's the best for the time being.